fastflix/widgets/panels/status_panel.py

```diff
@@ -111,6 +121,40 @@ def update_bitrate(self, bitrate):
     def update_title_bar(self):
         pass
 
+    def set_started_at(self, msg):
+        try:
+            started_at = datetime.datetime.fromisoformat(msg.split("__")[-1])
+        except Exception as e:
+            logger.warning(f"Unable to parse start time - {str(e)}")
```

Comment:
As this is a full on internal error if it happens, best to do `logging.exception` so it logs the traceback for us, then don't need to capture `e` at all.

Reply:
Yes that sounds very wise! Pushed in afd2d0b